notifications_utils/email_template.jinja2

```diff
@@ -104,6 +104,53 @@
     </tr>
   </table>
 {% endif %}
+{% if brand_banner %}
+  {% set brand_colour = brand_colour if brand_colour else '#0b0c0c' %}
+  <table role="presentation" width="100%" style="min-width: 100%;width: 100% !important;" cellpadding="0" cellspacing="0" border="0">
+      <tr>
+        <td width="100%" height="53" bgcolor="#{{brand_colour}}">
+          <!--[if (gte mso 9)|(IE)]>
+            <table role="presentation" width="580" align="center" cellpadding="0" cellspacing="0" border="0">
+              <tr>
+                <td>
+          <![endif]-->
+          <table role="presentation" width="100%" style="max-width: 580px;" cellpadding="0" cellspacing="0" border="0" align="center">
+            <tr>
+              {% if brand_logo %}
+                <td width="70" bgcolor="#{{brand_colour}}" valign="middle">
+                  <img src="{{ brand_logo }}" alt="" height="32" border="0" style="margin-top: 4px; margin-left: 10px;">
+                </td>
+              {% endif %}
+              {% if brand_name %}
+                <td width="100%" bgcolor="#{{brand_colour}}" valign="middle" align="left">
+                  <span style="
+                    display: block;
+                    padding-left: 5px;
+                    font-family: Helvetica, Arial, sans-serif;
+                    font-weight: 700;
+                    font-size: 19px;
+                    color: #ffffff;
+                    line-height: 25px;
+                    margin: 14px 0;
+                    {% if not brand_logo %}
```

Comment:
Should this not be `if brand_logo` – ie if there’s an image, add some space between it and the text?